env/scratch/cwagent-configmap.yaml

```diff
@@ -0,0 +1,71 @@
+kind: ConfigMap
+metadata:
+  name: prometheus-cwagentconfig
+  namespace: amazon-cloudwatch
+apiVersion: v1
+data:
+  # cwagent json config
+  cwagentconfig.json: |
```

Comment:
So this is the new CloudWatch agent to use for newer AWS kubernetes setup?

Reply:
This one and the none in the staging folder yep

Reply:
So no production as we test things first. Not a bad idea hehe.